x-pack/plugins/security_solution/public/timelines/components/row_renderers_browser/catalog/index.tsx

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLink } from '@elastic/eui';
+import React from 'react';
+import { ExternalLinkIcon } from '../../../../common/components/external_link_icon';
+
+import { RowRendererId } from '../../../../../common/types/timeline';
+import {
+  AuditdExample,
+  AuditdFileExample,
+  NetflowExample,
+  SuricataExample,
+  SystemExample,
+  SystemDnsExample,
+  SystemEndgameProcessExample,
+  SystemFileExample,
+  SystemFimExample,
+  SystemSecurityEventExample,
+  SystemSocketExample,
+  ZeekExample,
+} from '../examples';
+import * as i18n from './translations';
+
+const Link = ({ children, url }: { children: React.ReactNode; url: string }) => (
+  <EuiLink
+    href={url}
+    target="_blank"
+    rel="noopener nofollow noreferrer"
+    data-test-subj="externalLink"
+  >
+    {children}
+    <ExternalLinkIcon data-test-subj="externalLinkIcon" />
+  </EuiLink>
+);
+
+export interface RowRendererOption {
+  id: RowRendererId;
+  name: string;
+  description: React.ReactNode;
+  searchableDescription: string;
+  example: React.ReactNode;
+}
+
+export const renderers: RowRendererOption[] = [
+  {
+    id: RowRendererId.auditd,
+    name: i18n.AUDITD_NAME,
+    description: (
+      <span>
+        <Link url="https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat-module-auditd.html">
+          {i18n.AUDITD_NAME}
+        </Link>{' '}
+        {i18n.AUDITD_DESCRIPTION_PART1}
+      </span>
+    ),
+    example: AuditdExample,
```

Comment:
Consider changing the `destination.ip` in this example to be a `10.x.x.x` or `192.168.x.x`.  I'm not sure if it's related to this PR, but I also noticed that when hovering over `destination.ip` in the example, the tooltip that should read `destination.ip` is not displayed, but `destination.port` (next to it) is working as expected.